bpfd/src/server/bpf.rs

```diff
@@ -265,7 +312,7 @@ impl<'a> BpfManager<'a> {
             .unwrap()
             .values_mut()
             .collect::<Vec<&mut ExtensionProgram>>();
-        extensions.sort_by(|a, b| a.metadata.cmp(&b.metadata));
+        extensions.sort_by(|a, b| a.current_position.cmp(&b.current_position));
```

Comment:
I don't think we should sort by current position, as new programs would always go to the end, which isn't what we want if priority is lower.